conceptual/Npgsql/release-notes/6.0.md

```diff
@@ -168,6 +168,23 @@ The changes described above are far-reaching, and may break applications in vari
 AppContext.SetSwitch("Npgsql.EnableLegacyTimestampBehavior", true);
 ```
 
+### Parameter names now use case-sensitive matching
+
+Parameter names used to be case-insensitive:
+
+```c#
+using var command = new NpgsqlCommand("SELECT @foo", connection);
+command.Parameters.AddWithValue("FOO", 8);
+```
+
+For performance reasons, matching is now insensitive, so parameter names must exactly match ([#4027](https://github.com/npgsql/npgsql/issues/4027)).
```

Comment:
Pretty sure now it's case-sensitive , not insensitive.

Reply:
:facepalm: